setup.py

```diff
@@ -1103,7 +1103,9 @@ def get_extensions_from_sources(sources):
             'Topic :: Software Development :: User Interfaces'],
         dependency_links=[
             'https://github.com/kivy-garden/garden/archive/master.zip'],
-        install_requires=['Kivy-Garden>=0.1.4', 'docutils', 'pygments'],
+        install_requires=[
+            'Kivy-Garden>=0.1.4', 'docutils', 'pygments', 'oscpy'
```

Comment:
I wonder if that could be optionnal now? I know you use it in your daily job, just wondering.

Reply:
Sure, i guess it would be more convenient for platforms like android/ios where it's not useful.